Reproducible-Analytical-Pipelines-MVP.md

```diff
@@ -2,10 +2,10 @@
 
 ## Things to be aware of
 
-This guidance has been agreed upon by a working group of the [Reproducible Analytical Pipelines Champion](https://gss.civilservice.gov.uk/about-us/champion-networks/reproducible-analytical-pipeline-rap-champions/).
-It will be reviewed periodically by the RAP Champions and the Best Practice and Impact data science team.
+This guidance has been agreed by a working group of the [Reproducible Analytical Pipelines Champions](https://gss.civilservice.gov.uk/about-us/champion-networks/reproducible-analytical-pipeline-rap-champions/).
+It will be reviewed periodically by the RAP Champions and the Analysis Standards and Pipelines data science team in Methods and Quality Directorate the Office for National Statistics.
```

Comment:
Should this be *of* or *at* the Office for National Statistics? Also, is it Methods and Quality or Methodology and Quality?